README.md

```diff
@@ -6,21 +6,72 @@
 </p>
 
 <p align="center">
-  End-to-end testing framework for web3 developers building on EVM-compatible chains.
+  GuardianTest is an open-source, e2e testing framework for web3 developers building on EVM-compatible chains.
 <p>
 
 <br>
 
-## What the GuardianUI framework enables
+## Installation
+
+### Dependencies
+
+#### Installing Node.js
+
+* Check if you already have Node installed by opening a terminal or command prompt instance and executing `node -v`. You should see a version number in the return.
+* If you do not have Node already installed, go to [Node.js](https://nodejs.org/en/download) and select the version associated with your operating system.
+
+
+#### Installing Foundry
+
+* Check if you already have [Anvil](https://github.com/foundry-rs/foundry/tree/master/anvil) installed through Foundry by opening a terminal or command prompt instance and executing `anvil -V`. You should see a version number in the return. If you do not, follow one of the guides below.
+
+**Mac, Linux, and Windows**
+
+* [Install using Foundryup](https://book.getfoundry.sh/getting-started/installation#using-foundryup)
+* [Building from source](https://book.getfoundry.sh/getting-started/installation#building-from-source)
+
+**Docker**
+
+* [Installation for Docker](https://book.getfoundry.sh/getting-started/installation#using-foundry-with-docker)
+
+**CI**
+
+* [Installation for CI](https://book.getfoundry.sh/getting-started/installation#installing-for-ci-in-github-action)
+
+
+### Installing GuardianTest
+
+You can install GuardianTest using either npm or yarn:
+
+**npm**
+```bash
+npm install @guardianui/test
+```  
 
-- Testing in an environment similar to production across any EVM-compatible chain
-- Everything [Playwright](https://github.com/microsoft/playwright) already allows you to test (what is displayed, navigation works as expected, buttons are clickable, etc)
-- Making sure your frontend is creating the expected write smart contract interactions (transactions, approvals)
-- Ensuring your frontend responds to a completed transaction the way you are expecting
+**yarn**
+```bash
+yarn add @guardianui/test
+```
+
+### Installing Playwright Browsers
```

Comment:
add below this "You will need to have the latest Playwright browser drivers installed"